Samples/Android/Speech Devices SDK Starter App/example/app/src/main/java/com/microsoft/coginitiveservices/speech/samples/sdsdkstarterapp/MainActivity.java

```diff
@@ -27,29 +31,34 @@
 import com.microsoft.cognitiveservices.speech.translation.TranslationRecognizer;
 
 import java.io.File;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Calendar;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 
+import static com.microsoft.coginitiveservices.speech.samples.sdsdkstarterapp.LanguageCode.getCode;
 import static com.microsoft.cognitiveservices.speech.ResultReason.RecognizedKeyword;
 import static com.microsoft.cognitiveservices.speech.ResultReason.RecognizingSpeech;
 
+
 public class MainActivity extends AppCompatActivity {
 
     // Subscription
-    private static final String SpeechSubscriptionKey = "<enter your subscription info here>";
-    private static final String SpeechRegion = "westus"; // You can change this, if you want to test the intent, and your LUIS region is different.
-    private static final String LuisSubscriptionKey = "<enter your subscription info here>";
-    private static final String LuisRegion = "westus2"; // you can change this, if you want to test the intent, and your LUIS region is different.
-    private static final String LuisAppId = "<enter your LUIS AppId>";
+    private static String SpeechSubscriptionKey = "<enter your subscription info here>";
+    private static String SpeechRegion = "westus2"; // You can change this, if you want to test the intent, and your LUIS region is different.
+    private static String LuisSubscriptionKey = "<enter your subscription info here>";
+    private static String LuisRegion = "westus2"; // you can change this, if you want to test the intent, and your LUIS region is different.
+    private static String LuisAppId = "<enter your LUIS AppId>";
+
+    private static String Keyword = "computer";
+    private static String KeywordModel = "computer.zip";
 
-    private static final String Keyword = "Computer";
-    private static final String KeywordModel = "kws-computer.zip";
-    private static final String DeviceGeometry = "Circular6+1";
-    private static final String SelectedGeometry = "Circular6+1";
+    private static String DeviceGeometry = "<enter your microphone geometry>"; //"Circular6+1" or "Linear4"
+    private static String SelectedGeometry = "<enter your select geometry>"; //"Circular6+1" or "Linear4"
```

Comment:
selected geometry can be Circular6+1, Circular3+1, Linear4, Linear2

Reply:
I will  add Circular6+1  Circular3+1, Linear4, Linear2 in option menu